doc/source/user_guide/indexing.rst

```diff
@@ -584,45 +584,13 @@ without using a temporary variable.
    (bb.groupby(['year', 'team']).sum()
       .loc[lambda df: df['r'] > 100])
 
-.. _indexing.deprecate_ix:
```

Comment:
There is one reference to this label in v0.20.0.rst. Could you replace that with a harcoded link to an older version of the docs? Eg https://pandas.pydata.org/pandas-docs/version/1.0/user_guide/indexing.html#ix-indexer-is-deprecated

Reply:
Done.